### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by killian441**
Added more verbiage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,16 +19,16 @@ Properties
...
+-   **uniqueness**: Expression Property. If specified, each queue group (i.e. *group_by*) will not allow multiple signals that evaluate to the same *uniqueness*. If an incoming signal matches *group_by* and *uniqueness* with a signal already in the queue, then the new signal is dropped.
 
 Dependencies
 ----------------
 None
 
 Commands
 ----------------
-
+Use the nio_command block to issue the following commands:
```
Comment (hansmosh) [inline README.md]: Remove this line. That is one way to use the commands but it does not belong in the docs for each block. This kind of documentation would go well somewhere here: https://github.com/nioinnovation/docs

**Comment by hansmosh** [comment]
@killian441: Most of the updates look good. Thanks. Please remove that one line though, as it is not relevant to the block. Message me on Slack and I can clarify.

**Comment by killian441** [comment]
Updated per comments.

**Comment by hansmosh** [comment]
Thanks @killian441. Merged.

**Pull request merged**